### PR TITLE
PLAT-491: Test constructor in current pulse without object

### PR DESCRIPTION
**Pull request by MikhailZhukov**
**- What I did**
Add test for constructor with current pulse and without created object. Use new Journal.WaitStopOf functional for waiting execution done in tests

**- How I did it**
Write test

**- How to verify it**
Run test

**- Description for the changelog**

### CODE REVIEW

File: ledger-core/virtual/integration/constructor_test.go [hunk 1]
```diff
@@ -123,7 +126,94 @@ func TestVirtual_Constructor_WithExecutor(t *testing.T) {
...
+
+	pl := payload.VCallRequest{
+		CallType:       payload.CTConstructor,
+		CallFlags:      payload.BuildCallFlags(isolation.Interference, isolation.State),
+		CallAsOf:       server.GetPulse().PulseNumber,
+		Callee:         class,
+		CallSiteMethod: "test",
+		CallOutgoing:   outgoing,
+	}
+	msg := utils.NewRequestWrapper(server.GetPulse().PulseNumber, &pl).SetSender(server.JetCoordinatorMock.Me()).Finalize()
```
Comment (bigbes) [inline ledger-core/virtual/integration/constructor_test.go:179]: msg := server.WrapPayload(&pl).Finalize

Comment (MikhailZhukov) [reply ledger-core/virtual/integration/constructor_test.go]: Done

File: ledger-core/virtual/integration/constructor_test.go [hunk 2]
```diff
@@ -123,7 +125,94 @@ func TestVirtual_Constructor_WithExecutor(t *testing.T) {
...
+	}
+	msg := server.WrapPayload(&pl).Finalize()
+
+	typedChecker := server.PublisherMock.SetTypedChecker(ctx, mc, server)
+	typedChecker.VCallResult.Set(func(res *payload.VCallResult) bool {
+		require.Equal(t, res.ReturnArguments, runnerResult)
+		require.Equal(t, res.Callee, objectRef)
+		require.Equal(t, res.CallOutgoing, outgoing)
+
+		return false // no resend msg
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/constructor_test.go:186]: check CTMethod and CallFlags , too, please

Comment (MikhailZhukov) [reply ledger-core/virtual/integration/constructor_test.go:189]: done

Comment (MikhailZhukov) [reply ledger-core/virtual/integration/constructor_test.go:189]: done

**Pull request merged**